py/tests/algorithms/anomaly_likelihood_test.py

```diff
@@ -545,11 +550,12 @@ def testUpdateAnomalyLikelihoods(self):
     to estimate the distribution on fake data, followed by several calls
     to updateAnomalyLikelihoods.
     """
-
+    rnd = Random(self.GLOBAL_TEST_SEED)#we want different data for each generate
```

Comment:
actually, this resets the random state, but to the same GLOBAL_SEED. You probably want to either Random with a different seed, or leave be the old random state and just continue the sequence for rnd numbers ?

Reply:
...sorry, this is a new test,function, so it's Ok to call the Random()